src/components/App.tsx

```diff
@@ -0,0 +1,123 @@
+    image: AvengersPoster,
+    releaseDate: '2008',
+    runtime: '200 mins',
+    rating: '7.8',
+    overview:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam non feugiat felis, in accumsan justo. Quisque enim leo, volutpat ac tempus eu, ullamcorper eu nulla. Duis sagittis convallis dolor ac ullamcorper. Nam fringilla venenatis sem gravida vulputate. Vestibulum sagittis eu elit sed rhoncus. Etiam blandit erat a augue auctor, ut vestibulum ligula posuere. Sed arcu quam, tristique non accumsan a, eleifend sed lorem. In nec augue et ante maximus rhoncus. Fusce at neque lorem. Pellentesque a gravida dolor. Nam ac molestie tortor. Donec at nunc ut purus rhoncus fermentum in et tortor. Mauris pulvinar non ligula non pharetra. Nullam diam nulla, vehicula sit amet fringilla quis, sodales vel magna. Cras nec nulla tempor, ullamcorper purus a, egestas ex. Nam semper scelerisque arcu, a malesuada tortor.',
+  },
+  {
+    id: 5,
+    name: 'Inception',
+    genres: ['Action & Adventure'],
+    image: InceptionPoster,
+    releaseDate: '2008',
+    runtime: '200 mins',
+    rating: '7.8',
+    overview:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam non feugiat felis, in accumsan justo. Quisque enim leo, volutpat ac tempus eu, ullamcorper eu nulla. Duis sagittis convallis dolor ac ullamcorper. Nam fringilla venenatis sem gravida vulputate. Vestibulum sagittis eu elit sed rhoncus. Etiam blandit erat a augue auctor, ut vestibulum ligula posuere. Sed arcu quam, tristique non accumsan a, eleifend sed lorem. In nec augue et ante maximus rhoncus. Fusce at neque lorem. Pellentesque a gravida dolor. Nam ac molestie tortor. Donec at nunc ut purus rhoncus fermentum in et tortor. Mauris pulvinar non ligula non pharetra. Nullam diam nulla, vehicula sit amet fringilla quis, sodales vel magna. Cras nec nulla tempor, ullamcorper purus a, egestas ex. Nam semper scelerisque arcu, a malesuada tortor.',
+  },
+  {
+    id: 6,
+    name: 'Reservoir Dogs',
+    genres: ['Oscar winning Movie'],
+    image: ReservoirDogsPoster,
+    releaseDate: '2008',
+    runtime: '200 mins',
+    rating: '7.8',
+    overview:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam non feugiat felis, in accumsan justo. Quisque enim leo, volutpat ac tempus eu, ullamcorper eu nulla. Duis sagittis convallis dolor ac ullamcorper. Nam fringilla venenatis sem gravida vulputate. Vestibulum sagittis eu elit sed rhoncus. Etiam blandit erat a augue auctor, ut vestibulum ligula posuere. Sed arcu quam, tristique non accumsan a, eleifend sed lorem. In nec augue et ante maximus rhoncus. Fusce at neque lorem. Pellentesque a gravida dolor. Nam ac molestie tortor. Donec at nunc ut purus rhoncus fermentum in et tortor. Mauris pulvinar non ligula non pharetra. Nullam diam nulla, vehicula sit amet fringilla quis, sodales vel magna. Cras nec nulla tempor, ullamcorper purus a, egestas ex. Nam semper scelerisque arcu, a malesuada tortor.',
+  },
+];
```

Comment:
Starting from TS v4.9, we can use `satisfies` operator to define hardcoded data structures:
```ts
const yourObject = [
...
] satisfies {
  id: number;
  name: string;
  genres: GenreEnum[];
  image: string;
  releaseDate: Date;
  runtime: string;
  rating: number;
  overview: string;
}[];
```
With `satisfies` you get all typesafity with hints and TS understands further in the code, what your data structure is

Reply:
I fixed this in a later PR with a more conventional:

```
const initialState: { movies: Movies; } = {
  movies: [...]
}
```

But this is really handy to know for non-nested objects!